client/app/services/custom-button/_custom-button.sass

```diff
@@ -19,3 +19,14 @@
       border-color: $app-color-light-blue-4
       color: $app-color-medium-gray-2
       text-decoration: none
+.custom-button-action
+  &.disabled
```

Comment:
writing a new disabled style feels a lil queasy, why not just extend an existing disabled class?

Reply:
The sass work is to create a more specific selector that only applies to custom action buttons.  The specificity was needed to override some default classes that were applying styles